zipkin-storage/elasticsearch-http/src/test/java/zipkin/storage/elasticsearch/http/ElasticsearchHttpSpanConsumerTest.java

```diff
@@ -110,83 +110,60 @@ public void writesSpanNaturallyWhenNoTimestamp() throws Exception {
   }
 
   @Test
-  public void indexesServiceSpan_explicitTimestamp() throws Exception {
```

Comment:
Some existing tests have been changed for several reasons:

- related functionality ("servicespan" caching) eliminates repetitive writing of the same "servicespan" pairs to ES, therefore breaking some tests expectations
- tests order is important in a case of "servicespan" caching, so reordered existing tests in the file for better readability